src/Agent.Worker/DiagnosticLogManager.cs

```diff
@@ -198,6 +208,54 @@ public sealed class DiagnosticLogManager : AgentService, IDiagnosticLogManager
             executionContext.Debug("Diagnostic file upload complete.");
         }
 
+        // Dumping Agent Azure VM extension logs to the support files folder.
+        private void DumpAgentExtensionLogs(IExecutionContext executionContext, string supportFilesFolder, DateTime jobStartTimeUtc)
+        {
+            executionContext.Debug("Starting dumping agent extension logs.");
+
+            string pathToLogs = null;
+            string archiveName = null;
+            string timestamp = jobStartTimeUtc.ToString("yyyyMMdd-HHmmss");
+
+            if (PlatformUtil.RunningOnWindows)
+            {
+                string pathToVersions = "C:\\WindowsAzure\\Logs\\Plugins\\Microsoft.VisualStudio.Services.TeamServicesAgent";
```

Comment:
Could it make sense to extract hardcoded paths to the `internal static class`?

For reference: 
https://github.com/microsoft/azure-pipelines-agent/blob/3e4c9478b01e3bff5c5a15ca22bf45dc3fbe17f2/src/Agent.Worker/TaskDecoratorManager.cs#L89-L93